.github/labeler.yml

```diff
@@ -0,0 +1,28 @@
+# Use this to test the matches: http://www.globtester.com/
+
+# Any file within the config subfolder
+Config: 'config/**'
+
+# When the .DME is changed
+DME Edit: '**/*.dme'
+
+# Any .dmi changes
+Sprites: '**/*.dmi'
+
+# Javascript changes
+Javascript: '**/*.js'
+
+# Changes to a .dmm
+Map Change: '**/*.dmm'
+
+# Any changes to .ogg files are marked as sound
+Sound: '**/*.ogg'
+
+# Changes to the SQL subfolder
+SQL: 'SQL/**'
+
+# Encompasses both the tgui and the tgui-net folder
+tgui: '{tgui,tgui-next/**}'

```

Comment:
Maybe rename it to UI and add css and HTML files too

Reply:
but why